solution_architecture.md

```diff
@@ -210,27 +210,27 @@ The information listed above is not visible to the user, but is used internally
 As the values above are multiplied with each other, a single category with a risk score of 0 means that the overall risk score is also 0. 
 Additionally, a central threshold for the combined risk score specifies whether an exposure event should be considered or not. 
 Furthermore the Google/Apple framework allows to set a [```minimalRiskScore```](https://developer.apple.com/documentation/exposurenotification/enexposureconfiguration/3583692-minimumriskscore) to exclude exposure incidents with scores lower than the value of this property. 
 In the current version of the API the time spent within the ranges of attenuation buckets are accumulated over all exposure incidents during one matching session. 
 Since the number of requests is currently limited, it is not possible to get these values for each day and each exposure incident separately. 
-While by default there is no minimum value set, this value is being configured accordingly, so that resumably irrelevant exposure incidents are excluded. 
+While by default there is no minimum value set, this value is being configured accordingly, so that presumably irrelevant exposure incidents are excluded. 
 
 ![Figure 13: Calculation of the combined risk score](images/solution_architecture/figure_13.svg "Figure 13: Calculation of the combined risk score")
 
-Note that the transmission risk level plays a special role: It can be defined by the app and be associated with each individual diagnosis key (i.e. specific for each day of an infected person) that is being sent to the server. It contains a value from 1 to 8, which can be used to represent a calculated risk defined by the health authority and estimates the infectiousness and hence the likelihood of transmitting the disease.
+Note that the transmission risk level plays a special role in the above calculations: It can be defined by the app and be associated with each individual diagnosis key (i.e. specific for each day of an infected person) that is being sent to the server. It contains a value from 1 to 8, which can be used to represent a calculated risk defined by the health authority. As an example it could contain an estimate of the infectiousness of the potential infector at the time of contact and, hence, the likelihood of transmitting the disease. The specific values are defined as part of the [app](https://github.com/corona-warn-app/cwa-app-android/blob/ddb1f94f1d577f08863ae319c195d80cacf0bc3e/Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/ProtoFormatConverterExtensions.kt) - a motivation of the parameter choices is found in the document [Epidemiological Motivation of the Transmission Risk Level](https://github.com/corona-warn-app/cwa-documentation/blob/master/transmission_risk.pdf). 
```

Comment:
By linking to 
https://github.com/corona-warn-app/cwa-app-android/blob/ddb1f94f1d577f08863ae319c195d80cacf0bc3e/Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/ProtoFormatConverterExtensions.kt you are linking to a precise version (`ddb1f94f1d577f08863ae319c195d80cacf0bc3e`) of that file.

Is there a specific reason to link to a specific version of `ProtoFormatConverterExtensions.kt`? Wouldn't it be better to link to master?

https://github.com/corona-warn-app/cwa-app-android/blob/master/Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/ProtoFormatConverterExtensions.kt

```suggestion
Note that the transmission risk level plays a special role in the above calculations: It can be defined by the app and be associated with each individual diagnosis key (i.e. specific for each day of an infected person) that is being sent to the server. It contains a value from 1 to 8, which can be used to represent a calculated risk defined by the health authority. As an example it could contain an estimate of the infectiousness of the potential infector at the time of contact and, hence, the likelihood of transmitting the disease. The specific values are defined as part of the [app](https://github.com/corona-warn-app/cwa-app-android/blob/master/Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/ProtoFormatConverterExtensions.kt) - a motivation of the parameter choices is found in the document [Epidemiological Motivation of the Transmission Risk Level](https://github.com/corona-warn-app/cwa-documentation/blob/master/transmission_risk.pdf). 
```

Reply:
Thanks for your comments, that's actually a mistake, of course the link should have been to the main version.